var/spack/repos/builtin/packages/hohqmesh/package.py

```diff
@@ -18,6 +18,7 @@ class Hohqmesh(CMakePackage):
     license("MIT")
 
     version("main", branch="main")
+    version("1.5.4", sha256="d6e06873dd0db4205a62d317707da0be9f4f10b63430b4e1d43a4f266d7410ae")
```

Comment:
How was this calculated? For example, you can use the `spack checksum` command (https://spack.readthedocs.io/en/latest/command_index.html#spack-checksum).

I cannot confirm the sha256.  I get:

```suggestion
    version("1.5.4", sha256="8915ec3bc3c62b5a701ddaea2a8d39b8fba57cf9a0328bc3c29f2a589c800722")
```

Reply:
Manually using `sha256sum`:
```shell
?> curl -sSL https://github.com/trixi-framework/HOHQMesh/releases/download/v1.5.4/HOHQMesh-v1.5.4.tar.gz | sha256sum
d6e06873dd0db4205a62d317707da0be9f4f10b63430b4e1d43a4f266d7410ae  -
```
Did I do this wrong?

Reply:
Interesting that the releases archive URL you're using doesn't match the one implied by the `url` property; namely, `https://github.com/trixi-framework/HOHQMesh/archive/v1.5.4.tar.gz`.  If you use the derived URL (under the `archive` folder), you'll get the `sha256` I pasted above.

Reply:
Or you can run `spack checksum hohqmesh 1.5.4` to get the complete version directive containing the `sha256`.

Reply:
Ah, right. I think the difference is that for the release archive we pack the sources for a (small) dependency such that one does not require any additional sources. It thus totally makes sense that they differ... thanks for noticing.